src/main/resources/org/jenkinsci/plugins/workflow/job/WorkflowJob/sidepanel.jelly

```diff
@@ -31,14 +31,14 @@
             <j:set var="url" value="${h.getNearestAncestorUrl(request,it)}"/>
             <j:choose>
                 <j:when test="${it.parent==app}">
-                    <l:task icon="icon-up icon-md" href="${rootURL}/" title="${%Back to Dashboard}" contextMenu="false"/>
+                    <l:task icon="symbol-arrow-up" href="${rootURL}/" title="${%Back to Dashboard}" contextMenu="false"/>
```

Comment:
Why is this link present anyway? It links to the a page that can be accessed with the breadcrumbs.  It introduces clutter in the side panel. (The same is wrong in Jenkins core.) See discussion in https://github.com/jenkinsci/jenkins/pull/6783